cmd/wallets/transfer_funds.go

```diff
@@ -97,6 +106,21 @@ func RunTransferFunds(command *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	purpose, err := command.Flags().GetString("purpose")
+	if err != nil {
+		return err
+	}
+	beneficiaryJSON, err := command.Flags().GetString("beneficiary")
+	if err != nil {
+		return err
+	}
+	var beneficiary *uphold.Beneficiary
+	if len(beneficiaryJSON) > 0 {
+		err := json.Unmarshal([]byte(beneficiaryJSON), beneficiary)
```

Comment:
does this need to pass the mem address  json.Unmarshal([]byte(beneficiaryJSON), &beneficiary)

Reply:
beneficiary is already a pointer so I don't believe that's needed, cc @husobee

Reply:
might be wrong but I think the beneficiary would need to be initialised to pass the pointer otherwise its a zero-value pointer

Reply:
aha, yes good point